### ISSUE TITLE
intermittent Mac test-ref failure in ol_japanese_iroha_a.html

### ISSUE THREAD

**Issue by metajack** [servo/servo#7785]

**Comment by frewsxcv** [comment]
```
---- ol_japanese_iroha_a.html == ol_japanese_iroha_ref.html stdout ----
	thread 'ol_japanese_iroha_a.html == ol_japanese_iroha_ref.html' panicked at 'rendering difference: /tmp/servo-reftest-000243-diff.png', ../../tests/reftest.rs:372

stack backtrace:
   1:        0x1093d1110 - sys::backtrace::tracing::imp::write::h88885f5c2d394b3flbt
   2:        0x1093d0059 - panicking::on_panic::h7a7974779bbfef81TZw
   3:        0x1093b71e2 - sys_common::unwind::begin_unwind_inner::h879cfd5a30161a9evas
   4:        0x1093b76ce - sys_common::unwind::begin_unwind_fmt::h86e9e878cb969fb5B9r
   5:        0x109352266 - check_reftest::h8ef7b65ad5c9ab083Ta
   6:        0x109351407 - make_test::closure.16234
   7:        0x1093523d7 - boxed::F.FnBox<A>::call_box::h12986459124815722064
   8:        0x1093a228d - boxed::F.FnBox<A>::call_box::h5720435639045600348
   9:        0x1093a02b2 - sys_common::unwind::try::try_fn::h13271712310062512895
  10:        0x1093cf938 - __rust_try
  11:        0x1093c7550 - sys_common::unwind::try::inner_try::h30a7e5c7446717a0T6r
  12:        0x1093a0462 - boxed::F.FnBox<A>::call_box::h3379866730630249101
  13:        0x1093d266d - sys::thread::Thread::new::thread_start::h6e92c6728040a269Mxw
  14:     0x7fff8cbf6267 - _pthread_body
  15:     0x7fff8cbf61e4 - _pthread_start


failures:
    ol_japanese_iroha_a.html == ol_japanese_iroha_ref.html
```

**Comment by mbrubeck** [comment]
Looks like a platform-specific failure. These failures are all in mac-dev-ref-unit, though not all on the same build machine.

**Comment by Manishearth** [comment]
![servo-reftest-000118-diff](https://cloud.githubusercontent.com/assets/1617736/10652742/b4339f66-7876-11e5-8936-83a693e98c13.png)

is the diff, btw

**Comment by mbrubeck** [comment]
Appears to be fixed by #8257.
